METRICS.md

```diff
@@ -284,3 +284,69 @@ rate(firewood_space_from_end[5m])
 rate(firewood_proposal_commit{success="false"}[5m]) /
 rate(firewood_proposal_commit[5m])
 ```
+
+## Performance Tracking
+
+Firewood tracks its performance over time by running C-Chain reexecution benchmarks in AvalancheGo. This allows us to:
+
+- Monitor performance across commits and releases
+- Catch performance regressions early
+- Validate optimizations against real-world blockchain workloads
+
+Performance data is collected via the `Track Performance` workflow and published to GitHub Pages.
+
+### Running Benchmarks Locally
+
+Benchmarks can be triggered locally using just commands (requires nix):
+
+```bash
+# Run with defaults (current HEAD, master, default task/runner)
+just benchmark
+
+# With named parameters
```

Comment:
Readability here was strange. Would prefer the parameter descriptions first and then an examples section afterwards.